core/src/main/scala/org/ensime/core/Project.scala

```diff
@@ -69,6 +72,9 @@ class Project(
         // we could also just blindly send this on each connection.
         broadcaster ! Broadcaster.Persist(IndexerReadyEvent)
         log.debug(s"created $inserts and removed $deletes searchable rows")
+        val ensimeFlag = propOrNone("ensime.flag").getOrElse("")
+        if (ensimeFlag == "indexOnly")
+          context.parent ! CloseOnIndexUpdate
```

Comment:
`ShutdownRequest` is defined in the `server` module. I thought sending it would bring in a dependency for `core` on `server`. Please correct me if I am wrong.